kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -1522,6 +1551,22 @@ For each `CALL*` operation, we make a corresponding call to `#call` and a state-
            ...
          </account>
 
+    rule <k> #mkCreate ACCTFROM ACCTTO VALUE INITCODE
+          => #touchAccounts ACCTFROM ACCTTO ~> #accessAccounts ACCTFROM ACCTTO ~> #loadProgram INITCODE ~> #initVM ~> #execute
+         ...
+         </k>
+         <useGas> false </useGas>
+         <id> _ => ACCTTO </id>
+         <caller> _ => ACCTFROM </caller>
+         <callDepth> CD => CD +Int 1 </callDepth>
+         <callData> _ => .Bytes </callData>
+         <callValue> _ => VALUE </callValue>
+         <account>
+           <acctID> ACCTTO </acctID>
+           <nonce> NONCE => NONCE +Int 1 </nonce>
+           ...
+         </account>
+
```

Comment:
Same here, is it actually important that we duplicate this rule?

Reply:
Same as above.